core/arch/arm/kernel/entry_a32.S

```diff
@@ -433,6 +433,7 @@ shadow_no_access:
 	cmp	r0, r1
 	bls	shadow_no_access
 
+#if !defined(CFG_DYN_STACK_CONFIG)
```

Comment:
Note: to update to `CFG_DYN_CONFIG` (https://github.com/OP-TEE/optee_os/pull/7421 has been merged)